### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by tmarkovski**
Initial migration commit

Signed-off-by: Tomislav Markovski <tmarkovski@gmail.com>

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<Project>
+ <PropertyGroup>
+   <Version>4.0.0</Version>
+   <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <Company>AgentFramework</Company>
+    <Authors>Tomislav Markovski, Tobias Looker</Authors>
+    <PackageProjectUrl>https://github.com/streetcred-id/agent-framework</PackageProjectUrl>
```
Comment (tplooker) [inline Directory.Build.props]: Do we need to change this?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2018 Streetcred
```
Comment (tplooker) [inline LICENSE]: Whats the advice around the licence under hyperledger?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Agent Framework
+
+## .NET Core library for building Sovrin agents
+
+[![Build Status](https://dev.azure.com/streetcred/Agent%20Framework/_apis/build/status/Agent%20Framework%20-%20Build?branchName=master)](https://dev.azure.com/streetcred/Agent%20Framework/_build/latest?definitionId=10?branchName=master)
```
Comment (tplooker) [inline README.md]: Are we moving the CI at the same time?

File: docs/aspnetcore.rst [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+The default agent middleware is a simple implementation. You can `create your middleware
+<https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware/?view=aspnetcore-2.2>`_ and use that instead if you'd like to customize the message handling.
+
+.. code-block:: csharp
+
+    app.UseAgent<CustomAgentMiddlware>("http://localhost:5000/agent");
+
+See `AgentMiddleware.cs
+<https://github.com/streetcred-id/agent-framework/blob/master/src/AgentFramework.AspNetCore/Middleware/AgentMiddleware.cs>`_ for example implementation.
```
Comment (tplooker) [inline docs/aspnetcore.rst]: Shall we shift all these references too?

File: Directory.Build.props (hunk 1 above)
Comment (tmarkovski) [reply Directory.Build.props]: I assume Company needs to be Hyperledger. Not sure about authors, perhaps the same as Company? @ryjones Any thoughts on this?

File: LICENSE (hunk 2 above)
Comment (ryjones) [reply LICENSE]: I would replace this file with the stock Apache 2.0 file that GitHub provides

File: Directory.Build.props (hunk 1 above)
Comment (ryjones) [reply Directory.Build.props]: I would update the PackageProjectUrl but not worry about the Company right now

File: docs/aspnetcore.rst (hunk 4 above)
Comment (ryjones) [reply docs/aspnetcore.rst]: I'd fix that, as well, to point to the new repo

File: README.md (hunk 3 above)
Comment (ryjones) [reply README.md]: Up to you

Comment (tmarkovski) [reply README.md]: CI uses travis.ci which picked up right away. We will need to move CD and package deployment at some point. Package name spacing should also change from AgentFramework.* to Hyperledger.Aries.*
That means we have opportunity to reset the package version to 1.0. Let's have a separate discussion on that.

File: Directory.Build.props (hunk 1 above)
Comment (tmarkovski) [reply Directory.Build.props]: Updated all old url references, license with stock version, and removed binary files from history. We can squash and merge this PR.

**Comment by ryjones** [comment]
Once @tplooker approves and Travis finishes feel free to merge

**Pull request merged**